PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/function/runner/deploy/FunctionDeployConfiguration.java

```diff
@@ -149,7 +151,7 @@ public void setFunctionId(String id) {
     public void readExternal(Element element) throws InvalidDataException {
         this.functionDeployModel = Optional.ofNullable(element.getChild("FunctionDeployModel"))
                 .map(e -> XmlSerializer.deserialize(e, FunctionDeployModel.class))
-                .orElseGet(() -> Optional.ofNullable(element)
+                .orElseGet(() -> Optional.of(element)
```

Comment:
why change this line ? 
it's exactly sure that element is never null?

Reply:
Yes, defined in RunConfigurationBase interface, missed in our abstract parent class `public void writeExternal(@NotNull Element element) {`